### DISCUSSION TITLE
Are my wheels supposed to contain these include, lib, and share folder?

### DISCUSSION THREAD

**Discussion by jeffective** [wjakob/nanobind#580] [Q&A] [answered]
![image](https://github.com/wjakob/nanobind/assets/56497124/ecb47080-601e-4d0e-9d1b-3c1e7db2233c)

I think these may be coming from a separate library i have a dependency on that also uses CMake

**Comment by ianhbell** [comment, marked as answer]
In my case I have a CMake dependency (GEOS in my case) that installs headers and libraries and I have similar folders. If you don't want them, figure out how to convince the dependent library to not install its "thing" when pulled in in CMake. How to do that depends on the library causing this issue

**Reply by jeffective** [reply]
it looks like i was able to prevent the installation if the include / lib stuff using
https://stackoverflow.com/questions/64900981/how-do-you-prevent-cmake-from-install-ing-targets-from-within-projects-include

which meant adding the exlude from all option to add_subdirectory

```cmake
add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/ext/SOEM EXCLUDE_FROM_ALL)
```

**Reply by ianhbell** [reply]
That's a useful trick, I'll use it in my application as well
